### PR TITLE
eslint: configure import sorting rules

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**

### CODE REVIEW

File: .stylelintrc.json [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+        "scss/double-slash-comment-whitespace-inside": null,
+        "scss/no-global-function-names": null,
+        "scss/operator-no-unspaced": null,
+        "selector-class-pattern": null,
+        "selector-id-pattern": null,
 
-	"liberty/use-logical-spec": "always"
-  }
+        "liberty/use-logical-spec": "always"
+    }
```
Comment (jelly) [inline .stylelintrc.json:36]: I preferred the mess! Especially as every cockpit plugin does things different, different rules.

File: src/app.tsx [hunk 2]
```diff
@@ -19,25 +19,25 @@
...
 import React, { useContext, useEffect, useMemo, useState } from "react";
 
-import { Page, PageSection } from "@patternfly/react-core/dist/esm/components/Page";
-import { Sidebar, SidebarPanel, SidebarContent } from "@patternfly/react-core/dist/esm/components/Sidebar";
 import {
     AlertGroup, Alert, AlertVariant, AlertActionCloseButton
 } from "@patternfly/react-core/dist/esm/components/Alert";
+import { Page, PageSection } from "@patternfly/react-core/dist/esm/components/Page";
+import { Sidebar, SidebarPanel, SidebarContent } from "@patternfly/react-core/dist/esm/components/Sidebar";
 import { ExclamationCircleIcon } from "@patternfly/react-icons";
```
Comment (jelly) [inline src/app.tsx:27]: Hmm was this created by `--fix`? So it now sorts alphabetically?

File: src/app.tsx [hunk 3]
```diff
@@ -19,25 +19,25 @@
...
 
 import cockpit from "cockpit";
```
Comment (jelly) [inline src/app.tsx:29]: From my understanding, we prefer this to be separate. (Although it is imported from `pkg/lib` obviously.

File: src/header.tsx [hunk 4]
```diff
@@ -17,18 +17,19 @@
...
 
 import { CardHeader, CardTitle } from "@patternfly/react-core/dist/esm/components/Card";
 import { Divider } from "@patternfly/react-core/dist/esm/components/Divider";
-import { Flex } from "@patternfly/react-core/dist/esm/layouts/Flex";
 import { MenuToggle, MenuToggleAction } from "@patternfly/react-core/dist/esm/components/MenuToggle";
 import { SearchInput } from "@patternfly/react-core/dist/esm/components/SearchInput";
 import { Select, SelectList, SelectOption } from "@patternfly/react-core/dist/esm/components/Select";
 import { Text, TextContent, TextVariants } from "@patternfly/react-core/dist/esm/components/Text";
-import { SortByDirection } from '@patternfly/react-table';
+import { Flex } from "@patternfly/react-core/dist/esm/layouts/Flex";
```
Comment (jelly) [inline src/header.tsx:28]: Another eslint rule potential? :grin:

**Review by jelly — APPROVED** [review]
This is cool! But we would need to have this in all cockpit projects for consistency :cry:  

I'm happy this doesn't use regex in configuration to order the imports.

File: src/app.tsx (hunk 3 above)
Comment (jelly) [reply src/app.tsx:29]: Not a real blocker from my side, it does look nicer though.

Comment (martinpitt) [reply src/app.tsx:29]: I recently started to lump it together with other pkg/lib modules.

**Review by martinpitt — APPROVED** [review]
Thanks!

**Pull request merged**